usb-power-profiling.js

```diff
@@ -1289,7 +1272,41 @@ const app = (req, res) => {
     let counter = counterObject("USB power", deviceName, times,
                                 samples.slice(startIndex, endIndex)
                                        .map((sample, i) => Math.round(WattSecondToPicoWattHour(sample) * timeInterval(i))), true);
-    sendJSON(res, [counter], true);
+
+    return [counter];
+}
+
+async function resetPowerData() {
+    //TODO: include data for all devices
```

Comment:
Might be as easy as `for (let device of gDevices) {`

Reply:
If we want to be smarter, we can add `gDevices = gDevices.filter(d => !!d.device);` to remove the devices that have been detached.

Reply:
The filter was causing some odd things to happen with the logging (more failed to find endpoints logs) so I didn't add it.